test/preprocessing/geometries/geometries.jl

```diff
@@ -0,0 +1,186 @@
+
+        @testset "Rotation Angle $(rot_angle)" for rot_angle in 0.0:(pi / 8):(2pi)
+            v1 = rot([0.0, 0.0], rot_angle)
+            v2 = rot([0.0, 0.5], rot_angle)
+            v3 = rot([1.0, 0.5], rot_angle)
+            v4 = rot([1.0, 0.0], rot_angle)
+
+            points_rectangular_clockwise = [v1 v2 v3 v4 v1]
+            points_rectangular_counter_clockwise = [v1 v4 v3 v2 v1]
+
+            edge_vertices = [(v1, v2), (v2, v3), (v3, v4), (v4, v1)]
+
+            edge_normals = [
+                rot([-1.0, 0.0], rot_angle),
+                rot([0.0, 1.0], rot_angle),
+                rot([1.0, 0.0], rot_angle),
+                rot([0.0, -1.0], rot_angle),
+            ]
+
+            sqrt2 = sqrt(2)
+            vertex_normals = [
+                rot([[-sqrt2 / 2, -sqrt2 / 2], [-sqrt2 / 2, sqrt2 / 2]], rot_angle), # edge 1
+                rot([[-sqrt2 / 2, sqrt2 / 2], [sqrt2 / 2, sqrt2 / 2]], rot_angle),   # edge 2
+                rot([[sqrt2 / 2, sqrt2 / 2], [sqrt2 / 2, -sqrt2 / 2]], rot_angle),   # edge 3
+                rot([[sqrt2 / 2, -sqrt2 / 2], [-sqrt2 / 2, -sqrt2 / 2]], rot_angle), # edge 4
+            ]
+
+            geometry_clockwise = TrixiParticles.Polygon(points_rectangular_clockwise)
+
+            for edge in eachindex(edge_vertices)
+                @test all(isapprox.(geometry_clockwise.edge_vertices[edge],
+                                    edge_vertices[edge], atol=1e-14))
+            end
+            for vertex in eachindex(vertex_normals)
+                @test all(isapprox.(geometry_clockwise.vertex_normals[vertex],
+                                    vertex_normals[vertex], atol=1e-14))
+            end
+            @test all(isapprox(geometry_clockwise.edge_normals, edge_normals, atol=1e-14))
+
+            geometry_cclockwise = TrixiParticles.Polygon(Matrix(points_rectangular_counter_clockwise))
+
+            for edge in eachindex(edge_vertices)
+                @test all(isapprox.(geometry_cclockwise.edge_vertices[edge],
+                                    edge_vertices[edge], atol=1e-14))
+            end
+            for vertex in eachindex(vertex_normals)
+                @test all(isapprox.(geometry_cclockwise.vertex_normals[vertex],
+                                    vertex_normals[vertex], atol=1e-14))
+            end
+            @test all(isapprox(geometry_cclockwise.edge_normals, edge_normals, atol=1e-14))
+        end
+    end
+
+    @testset verbose=true "Real World Data" begin
+        data_dir = pkgdir(TrixiParticles, "examples", "preprocessing", "data")
+
+        @testset verbose=true "2D" begin
+            files = ["hexagon", "circle", "inverted_open_curve"]
+            n_edges = [6, 63, 240]
+
+            @testset "Test File `$(files[i])`" for i in eachindex(files)
+                # Checked in ParaView with `trixi2vtk(geometry)`
+                data = TrixiParticles.CSV.read(joinpath(validation_dir(), "preprocessing",
```

Comment:
The validation directory is for validation of simulation results against literature experiments. Can you please move these files to `test/`?